integrations/tensorflow/iree_tf_compiler/TFL/test/retain_call_once_funcs.mlir

```diff
@@ -0,0 +1,21 @@
+// RUN: iree-opt-tflite -allow-unregistered-dialect -split-input-file -pass-pipeline='iree-tflite-retain-call-once-funcs' %s | IreeFileCheck %s
```

Comment:
Does this actually need `-allow-unregistered-dialect`?

```suggestion
// RUN: iree-opt-tflite -split-input-file -pass-pipeline='iree-tflite-retain-call-once-funcs' %s | IreeFileCheck %s
```

Reply:
Unfortunately yes. Because of the !tf.resource type we need to support just ignoring the unknown type.

Reply:
I'm sure the answer is "because TF", but why is a `tfl` op using a `tf_type`?

If that dependency is really wired up that way, then why do we have a `iree-opt-tflite` tool and `iree_tf_compiler/TFL` directory at all? There was another comment thread about potentially unifying them.

Short of a full merging or upstream refactoring, can the tool at least register the dialects it needs?